flasher.cpp

```diff
@@ -481,6 +483,33 @@ void Flasher::GetVersion()
     emit ShowTextInBrowser(serial_port_.readAll());
 }
 
+bool Flasher::GetVersionJson(QJsonObject& out_json_object)
+{
+    bool success = false;
+    QByteArray out_data;
+    if(ReadMessageWithCrc(kVersionJsonCmd, sizeof(kVersionJsonCmd), kSerialTimeoutInMs, out_data)) {
+        QJsonDocument out_json_document = QJsonDocument::fromJson(QString(out_data).toUtf8());
+        out_json_object = out_json_document.object();
+
+        QString git_info;
+        git_info = "Git branch: ";
+        git_info.append(out_json_object.value("git_branch").toString());
+        emit ShowTextInBrowser(git_info);
+
+        git_info = "Git hash: ";
+        git_info.append(out_json_object.value("git_hash").toString());
+        emit ShowTextInBrowser(git_info);
+
+        git_info = "Git tag: ";
+        git_info.append(out_json_object.value("git_tag").toString());
+        emit ShowTextInBrowser(git_info);
```

Comment:
We can configure whole message and then show it.
```suggestion
        QString git_info = "Git branch: ";
        git_info.append(out_json_object.value("git_branch").toString());
        git_info.append("\nGit hash: ");
        git_info.append(out_json_object.value("git_hash").toString());
        git_info.append("\nGit tag: ");
        git_info.append(out_json_object.value("git_tag").toString());
        emit ShowTextInBrowser(git_info);
```

Reply:
That was also my first thought, but because of taking care of new lines and containing all the strings, somehow this occurred as a more elegant solution. If you have an idea how to make it more pretty, let me know.